src/utils/options.js

```diff
@@ -33,3 +34,16 @@ export function toFilterOpts(flags: Flags) {
 
   return filterOpts;
 }
+
+/**
+ * Takes a string in one of the following forms:
+ *  "pkgName", "pkgName@version", "@scope/pkgName", "@scope/pkgName@version"
+ * and returns an object with the package name and version (if passed)
+ */
+export function toDependency(dependencyString: string): Dependency {
```

Comment:
Is this an okay place for this helper? It's used for converting a certain *kind* of flag to be typed correctly